pkg/api/testdata/testdata.json

```diff
@@ -22,7 +22,7 @@
       "isForcedVariationEnabled": false,
       "key": "CAMPAIGN_1",
       "variations": {},
-      "type": "FEATURE_TEST"
+      "type": "FEATURE_ROLLOUT"
```

Comment:
you are changing the test data, what about the test case?

Reply:
Checked